vault/resource_raft_autopilot_test.go

```diff
@@ -5,15 +5,17 @@ import (
 	"testing"
 
 	"fmt"
+
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-vault/util"
 	"github.com/hashicorp/vault/api"
 )
 
 func TestAccRaftAutopilotConfig_basic(t *testing.T) {
 	resource.Test(t, resource.TestCase{
 		Providers:    testProviders,
-		PreCheck:     func() { testAccPreCheck(t) },
+		PreCheck:     func() { util.TestRaftPreCheck(t) },
```

Comment:
Think this might work, although it has not been tested.
```suggestion
		PreCheck:     func() { 
				util.TestAccPreCheck(t)
				if _, ok ; os.LookupEnv("SKIP_RAFT_TESTS"); ok {
					t.Skip("Warning: SKIP_RAFT_TESTS set, skipping test")
			},
		},
```

Reply:
Tested that it works and fixed!